features/steps/core.py

```diff
@@ -180,9 +187,9 @@ def check_output(context, text=None):
 def check_output_time_inline(context, text):
     out = context.stdout_capture.getvalue()
     local_tz = tzlocal.get_localzone()
-    utc_time = date_parser.parse(text)
-    local_date = utc_time.astimezone(local_tz).strftime("%Y-%m-%d %H:%M")
-    assert local_date in out, local_date
+    date, flag = CALENDAR.parse(text)
+    output_date = time.strftime("%Y-%m-%d %H:%M",date)
+    assert output_date in out, output_date
```

Comment:
Did we mean to put this back? This was removed in earlier work. Is there a reason this needs to go back?

Reply:
See above comment.